bin/rucio-conveyor-stager

```diff
@@ -34,8 +34,7 @@ def get_parser():
     """
     Returns the argparse parser.
     """
-    parser = argparse.ArgumentParser()
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(description="Conveyor is a daemon to manage file transfers. The conveyor-stager is responsible for make transfers from Tape endopoints to stage endpoints. This stage area is used as a buffer controlled by Rucio, in the sense that Rucio have control over the retention policy of the files in the staging area.")  # noqa E501
```

Comment:
The conveyor-stager is responsible for issuing staging requests to the tape system. 
Skip the rest